src/cli/add_test.py

```diff
@@ -0,0 +1,72 @@
+"""Integration and function testing for add.py"""
+
+from pathlib import Path
+from unittest.mock import MagicMock, patch
+
+from click.testing import CliRunner
+
+from src.api.wrappers.cf_wrapper import CurseForgeWrapper
+from src.cli.add import INVALID_FILE_ID_ERROR_MSG, Add
+from src.config.lockfile_entry import LockfileEntry
+from src.lib.asset import CurseForgeAsset
+from src.util.enum import AssetType, Platform, Side
+
+
+@patch('src.config.lockfile.LOCKFILE_FILENAME', 'test_m3.lock.json')
```

Comment:
Exercise 1: Define your own decorator that does this, so we can decorate with @patch_m3_jsons instead of calling @patch on the json files.